cpp/example/super_capacitor.info

```diff
@@ -0,0 +1,99 @@
+type SuperCapacitor
+dim  2
+verbosity 1
```

Comment:
we'll have to think of a strategy to keep them in sync. one option would be to only define the geometry in that input and overwrite the node in the tree. you don't have to worry about it right now though.